controlpanel/frontend/jinja2/tool-list.html

```diff
@@ -38,11 +38,14 @@ <h1 class="govuk-heading-xl">Your tools</h1>
            target="_blank">
           Open
         </a>
-        <a class="govuk-button hmcts-button--secondary tool-action {% if not deployment %} govuk-visually-hidden {% endif %}"
-           data-action-name="restart"
-           href="{{ url('restart-tool', kwargs={'name': tool.chart_name}) }}">
-          Reset
-        </a>
+        <form action="{{ url('restart-tool', kwargs={'name': tool.chart_name}) }}"
+              class="govuk-button hmcts-button--secondary tool-action {% if not deployment %} govuk-visually-hidden {% endif %}"
+              data-action-name="restart"
+              method="post">
+          {{ csrf_input }}
+          {# <input type="hidden" value="{{ tool.version }}" name="version"> #}
+          <button class="govuk-button hmcts-button--secondary right">Reset</button>
```

Comment:
I vote to keep this `Restart` or users will confuse this button with "rest the environment to factory" (e.g. delete conda envs etc...)

But nice one, this should fix the restart button `405` error 👍

Reply:
Sorry, copy-pasta here - class names need tidying up

Reply:
Oh, I see this is where the "Reset" caption comes from.